dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -649,6 +665,14 @@ def self.csv_header(course)
       end
     end
 
+    # @override
+    def self.cohort_csv_header
+      CSV.generate do |csv|
+        csv << ['Date Accepted', 'Applicant Name', 'District Name', 'School Name',
+                'Email', 'Assigned Workshop', 'Registered Workshop']
+      end
+    end
```

Comment:
I'm not crazy about having multiple csv presenters as methods in the class itself. This is fine for now, but we should consider using something like the serializers for csv formatting as a future cleanup or if we end up needing even more formats.